core/templates/pages/exploration-player-page/services/exploration-player-state.service.ts

```diff
@@ -257,8 +255,7 @@ export class ExplorationPlayerStateService {
       }, featuresData);
       this.explorationEngineService.init(
         explorationData, null, null, null, null, [], callback);
-      this.playerCorrectnessFeedbackEnabledService.init(
-        explorationData.correctness_feedback_enabled);
+      this.playerCorrectnessFeedbackEnabledService.init(true);
```

Comment:
Ditto.

Reply:
Dropped the service as it was unnecessary